lib/mayaUsd/base/CMakeLists.txt

```diff
@@ -6,27 +6,27 @@ target_sources(${PROJECT_NAME}
         debugCodes.cpp
 )
 
-set(headers
+set(HEADERS
     api.h
     debugCodes.h
 )
 
 # -----------------------------------------------------------------------------
-# promoted headers
+# promoted HEADERS
```

Comment:
Again comment should not be uppercase

Reply:
Addressed 07d962479409f7375694679beed213da71859b0c